content/user/admin_pages/localization/languages.md

```diff
@@ -1,59 +1,82 @@
-
-Check this box if this is the default language for this site.
-
-When you finish entering data, click on the _insert_ button to add the new language. Click on the _cancel_ button to discard this form.
-
-## Edit Language
-
-If you need to change the information about an existing language, click on the named language so that the right arrow appears in the Action column, then click on the _edit_ button. The fields on the screen are the same for adding a language. When finished editing, click on the _update_ button to save the changes, or click on _cancel_ to discard them.
-
-## Delete Language
-
-To remove a language, click on the named language so that the right arrow appears in the Action column, then click on the _Delete_ button. If this is not the default language, you will be asked to confirm. Click on the _delete_ button again to delete the language, or _cancel_ if you do not wish to delete this language. If this is the default language, only the _cancel_ button will appear, as you cannot delete the default language.
-
-## Installing a Language Pack
-
-Once you have added a new language, you will need to get the language pack files that 
-correspond to that language.  Language packs may be found in 
-the [Zen Cart Downloads](https://www.zen-cart.com/downloads.php?do=cat&id=6).
-
-Language packs are structured to correspond to your cart folders, so you can just 
-rename the admin folder if one exists and then upload all the files. 
-
-If you're not familiar how to upload a Zen Cart plugin, follow the instructions on 
-[how to install a plugin](/user/plugins/how_to_install_a_plugin/). 
-
+---
+title: Languages
+description: Zen Cart Languages Admin Page 
+category: admin_pages
+weight: 20
+---
+
+This page allows you to add additional language(s) in order to internationalize the output generated by Zen Cart.
+## New Language
+
+To add a new language, click on the _new language_ button. A form will appear. Enter the information, then click on the _insert_ button to add the new language. Click on the _cancel_ button if you change your mind.
+
+### Name
+
+This is the name of the language, probably best as expressed _in itself_, e.g. Francais for French, Español for Spanish, Deutsch for German, etc.
+
+### Code
+
+This is the two-letter ISO 639 two-letter language code, e.g. en for English, fr for Francais, es for Español, etc. If you don't know the code, you can find it on Wikipedia [here](http://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).
+
+### Image
+
+This would be a small icon used to identify the language. This would most likely be a gif image of the typical flag for some country commonly associated with that language. This should be located in the `/includes/languages/YOURLANGUAGE/images` folder.
+
+### Directory
+
+This directory is where the translated php source files for this language are contained. For example, enter '*english*' the directory for the English language is located at 
+`/includes/languages/english`. 

```

Comment:
This sentence rewording feels awkward, especially given the content on the unchanged line.